src/guide/build.md

```diff
@@ -0,0 +1,70 @@
+# Build Iroha 2 Client
+
+After you [installed Iroha](./install.md) from GitHub, follow these
+instructions to build Iroha 2 client:
+
+1. [Install the Rust Toolchain](#install-the-rust-toolchain)
+2. [Build Iroha 2 Client](#build-iroha-client)
+
+## Install the Rust Toolchain
+
+You need
+[a working Rust toolchain](https://www.rust-lang.org/learn/get-started):
+`cargo`, `rustc` v1.60 and up. [^1]
+
+[^1]:
+    If you're having issues with installing Rust compatible with our code
+    (2021 edition), please consult the
+    [troubleshooting](#troubleshooting-rust-toolchain) section.
+
+The installation of Rust Toolchain is normally a straightforward process,
+but we've added
+[troubleshooting](./troubleshooting/installation-issues.md#troubleshooting-rust-toolchain)
+details for each stage in case you experience issues with the installation
+process.
+
+The easiest way to get the official `rustup` script is to run:
+
+```bash
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+```
+
+Or, alternatively, you can install `rustup` via your operating system’s
+package manager.
+
+::: tip
+
+If you know what you're doing, you can also install the Rust toolchain
+directly, without `rustup`.
+
+:::
+
+If you go with the one-line `curl` script, you will be guided through the
```

Comment:
```suggestion
If you chose to use the one-line `curl` script, you will be guided through the
```

Reply:
Made this change because "go with" in two consecutive sentences is a bit repetitive